### ISSUE TITLE
updating TypesScript from 2.1 to 2.5

### ISSUE THREAD

**Issue by JLarky** [brunch/typescript-brunch#42]
I decided to write this up, because it wasn't clear for me how to do it and maybe this will be helpful for other people.

If you are using npm 5.0 with package-lock.json or shrinkwrap you can update to later versions of typescript without having to use forks.

Principle described [here](https://nodejs.org/en/blog/npm/managing-node-js-dependencies-with-shrinkwrap/), and here's example in case `package-lock.json` file:

```
    "brunch-typescript": {
      "version": "2.1.0",
      "resolved": "https://registry.npmjs.org/brunch-typescript/-/brunch-typescript-2.1.0.tgz",
      "integrity": "sha1-CbJJHp2mxm4o+6udxIxmFhsL6xg=",
      "dev": true,
      "requires": {
        "anymatch": "1.3.0",
        "typescript": "https://registry.npmjs.org/typescript/-/typescript-2.5.2.tgz"
      },
      "dependencies": {
        "typescript": {
          "version": "https://registry.npmjs.org/typescript/-/typescript-2.5.2.tgz",
          "integrity": "sha1-8PBF4Zb2mnLwayX9O9OdAcPOmYQ=",
          "dev": true
        }
      }
    },
```

just find `"brunch-typescript"` section in your file and change `"2.1.6"` to `"https://registry.npmjs.org/typescript/-/typescript-2.5.2.tgz"`, then run `npm install`.

**Comment by paulmedynski** [comment]
This should be solved with #43.

**Comment by paulmedynski** [comment]
Hi @JLarky - can you confirm if the fix in #43 solves your problem?  If so, please close this issue.

**Comment by JLarky** [comment]
I presume that it did :) I do not use brunch anymore
